src/Sulu/Bundle/CategoryBundle/Command/RecoverCommand.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+namespace Sulu\Bundle\CategoryBundle\Command;
+
+use Doctrine\DBAL\Driver\PDOException;
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\Query\ResultSetMapping;
+use Sulu\Bundle\CategoryBundle\Entity\CategoryRepository;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @package Sulu\Bundle\CategoryBundle\Command
+ */
+class RecoverCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this->setName('sulu:categories:recover')
+            ->addOption(
+                'force',
+                'f',
+                InputOption::VALUE_NONE,
+                'update tree'
+            )
+            ->addOption(
+                'depth',
+                'd',
+                InputOption::VALUE_NONE,
+                'fix depth'
```

Comment:
What does this do? I think the option should be called `fix-depth` and the description should be more descriptive :)